src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  setSelectedTodoId: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setSelectedTodoId }) => {
```

Comment:
The naming convention for the event handler `setSelectedTodoId` does not follow the recommended practices. A better name could be `handleSetSelectedTodoId` to reflect that it is an event handler function. Please refer to the recommended naming conventions [here](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364).